### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by ashmckenzie** [AtomLinter/linter-rubocop#36]
[Enter steps to reproduce below:]

1. Started atom with `--one`
2. Open any Ruby file

**Atom Version**: 0.194.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [linter-rubocop](https://github.com/AtomLinter/linter-rubocop) package, v0.2.2

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/ash/.atom/packages/linter-rubocop/lib/linter-rubocop.coffee:36

TypeError: undefined is not a function
  at LinterRubocop.destroy (/Users/ash/.atom/packages/linter-rubocop/lib/linter-rubocop.coffee:36:17)
  at LinterView.remove (/Users/ash/.atom/packages/linter/lib/linter-view.coffee:252:7)
  at /Users/ash/.atom/packages/linter/lib/linter-view.coffee:124:8
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:279:27)
  at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:45:58)
  at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:444:18)
  at Pane.module.exports.Pane.destroyActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:419:12)
  at Workspace.module.exports.Workspace.destroyActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:582:35)
  at Workspace.module.exports.Workspace.destroyActivePaneItemOrEmptyPane (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:621:21)
  at atom-workspace.atom.commands.add.core:close (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace-element.js:307:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:178:20)

```

### Commands

```
     -1:16.8.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
  2x -0:08.1.0 core:close (atom-text-editor.editor.vim-mode.command-mode.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "atom-spark-core",
      "welcome",
      "save-session",
      "preview",
      "color-picker"
    ],
    "themes": [
      "atom-dark-ui",
      "base16-tomorrow-dark-theme"
    ],
    "audioBeep": false,
    "followSymlinks": true,
    "projectHome": "/Users/ash/personal"
  }
}
```

### Installed Packages

```coffee
# User
Atom-Syntax-highlighting-for-Sass, v0.5.0
angularjs, v0.1.0
atom-alignment, v0.12.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.12.0
editorconfig, v1.0.0
file-icons, v1.5.4
git-tab-status, v1.7.0
highlight-line, v0.10.2
highlight-selected, v0.9.2
jsonlint, v1.1.0
language-arduino, v0.2.1
language-docker, v1.1.3
language-haml, v0.19.0
language-ini, v1.10.0
language-salt, v0.2.0
linter, v0.12.1
linter-rubocop, v0.2.2
linter-ruby, v0.1.4
minimap, v4.7.6
minimap-color-highlight, v4.1.0
minimap-find-and-replace, v4.2.0
minimap-git-diff, v4.1.2
minimap-highlight-selected, v4.2.1
minimap-selection, v4.2.0
open-in-sourcetree, v0.1.2
open-last-project, v0.4.1
pretty-json, v0.3.3
project-manager, v1.15.5
rspec, v0.1.9
set-syntax, v0.2.1
symbols-tree-view, v0.9.2
vim-mode, v0.44.0

# Dev
No dev packages
```

**Comment by nicolasmccurdy** [comment]
This should be fixed before Atom 1.0 is released.

**Comment by caseywebdev** [comment]
I believe this is no longer an issue.
